api/api.py

```diff
@@ -193,6 +196,12 @@ def prefix(path, routes):
 
         # Sessions
 
+        prefix('/sessions', [
+            route('/<cid:{cid}>/jobs',          ContainerHandler, h='get_jobs',     m=['GET']),
+            route('/<cid:{cid}>/subject',       ContainerHandler, h='get_subject',  m=['GET']),
+        ]),
+
+        route('/sessions/<cid:{cid}>/jobs',    ContainerHandler, h='get_jobs', m=['GET']),
```

Comment:
Are these new routes entirely? They don't feel like it, but I can't see where they moved from.